packages/auth/e2e/user.e2e.js

```diff
@@ -165,7 +319,7 @@ describe('auth().currentUser', () => {
       } catch (error) {
         // Reject
         try {
-          await firebase.auth().currentUser.delete();
+          // await firebase.auth().currentUser.delete();
```

Comment:
Think this line was still needed as if `sendEmailVerification` throws in the `try` block then `delete` won't get called